Automattic/jetpack
# renovate: Fix ignorePaths config

#### Changes proposed in this Pull Request:

The default ignorePaths ignores all files in `/tests/` directories,
which includes our `projects/plugins/*/tests/e2e/package.json`.

It turns out adjusting this configuration can't be done from the global
config in renovate-config.js, as the preset expansion overwrites it no
matter what I try. But we should be able to do it by adding back a
`renovate.json5` repo-level config file.

Most of the configuration stays in the global config, because that can
do things like read the list of all packages.

This also adds `.json5` as a recognized extension for the prettier
invocations, since it wasn't there already.

#### Other information:


#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Fork the repo, merge this there, adjust the renovate config so it'll run in the fork, run renovate, and see that it picks up the e2e package.json files.
  * :heavy_check_mark:  https://github.com/anomiex/jetpack/runs/8162699653?check_suite_focus=true#step:5:820
* Merge this, then have renovate regenerate #25974. It should update the e2e package.json files too.
